assets/styles/style.css

```diff
@@ -0,0 +1,100 @@
+/* //poppins fonts */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+/* global variables  */
+:root {
+  --mobile-margin-x: 30px;
+  --primary-color: #6070ff;
+  --primary-font: 'Poppins', sans-serif;
+  --primary-dark-color: #172b4d;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  box-sizing: border-box;
+  font-family: sans-serif;
+  background-color: rgba(234, 234, 255, 0.433);
```

Comment:
- [ ] A major requirement for this project is to make your design to be exactly like the Figma template design as much as possible. I would suggest that you add the proper background image in the section as seen in your chosen template design. 

> Figma design
<img width="222" alt="Screenshot 2023-04-27 at 16 17 01" src="https://user-images.githubusercontent.com/71140133/234908387-587b3644-38f0-4334-8b9d-c959de250c25.png">


> Your design
<img width="356" alt="Screenshot 2023-04-27 at 16 17 34" src="https://user-images.githubusercontent.com/71140133/234908517-b7e9b507-239b-43c1-9f4b-3711e38f3b3d.png">